cmd/secrets/secret_helper.go

```diff
@@ -53,105 +81,112 @@ type secretsRequest struct {
 	Secrets []string `json:"secrets"`
 }
 
-// ReadSecrets implements a secrets reader from a directory/mount
-func readSecrets(r io.Reader, w io.Writer, dir string) error {
-	in, err := ioutil.ReadAll(r)
+type backend interface {
+	// secretID has a different format on each secret backend. In the file
+	// backend, it's a path, whereas in the kubernetes secret one it has the
+	// following format: "namespace/name/key".
+	get(secretID string) s.Secret
```

Comment:
nit: Since this is an interface, I would assume that all methods would be public

Reply:
Yes. Same as https://github.com/DataDog/datadog-agent/pull/9132#discussion_r707657274